### PR TITLE
APP-4689 feat(dropdown): add small styles for dropdown

### PR DESCRIPTION
**Pull request by AliciaSymphony**
## Ticket
https://perzoinc.atlassian.net/browse/APP-4689

## Description
Override jss

## Demo
Screenshot 2021-12-02 at 13 03 06

PR Styles: https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/pull/179

### CODE REVIEW

**Review by symphony-annaLlorens — APPROVED** [review]

File: src/components/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -192,7 +192,8 @@ export class Dropdown<T = LabelValue> extends React.Component<
             menuPortal: (base: CSSProperties) => ({ ...base, ...menuPortalStyles }),
             valueContainer: (base: CSSProperties) => ({
               ...base, maxHeight: `${maxHeight}px`
-            })
+            }),
+            input: (size === 'small') ? (base: CSSProperties) => ({...base, margin: '0 2px'}) : undefined,
```
Comment (symphony-jean-michael) [inline src/components/dropdown/Dropdown.tsx:196]: Why do you need to use some inline style? Could you please use a CSS class instead?

Comment (AliciaSymphony) [reply src/components/dropdown/Dropdown.tsx:196]: No we can't, the issue is explained here https://perzoinc.atlassian.net/browse/APP-4689

Comment (symphony-jean-michael) [reply src/components/dropdown/Dropdown.tsx:196]: Ho! I see sorry! ^^

**Review by symphony-jean-michael — APPROVED** [review]
LGTM

**Pull request merged**